trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py

```diff
@@ -14,7 +14,7 @@
 from SUAVE.Methods.Geometry.Three_Dimensional import  orientation_product, orientation_transpose
 
```

Comment:
update header with name and date when modified

Reply:
Done